include/Foundation/NSKeyValueCoding.h

```diff
@@ -48,16 +48,16 @@ FOUNDATION_EXPORT NSString* const NSUnionOfSetsKeyValueOperator;
 - (id)valueForUndefinedKey:(NSString*)key;
 - (NSMutableArray*)mutableArrayValueForKey:(NSString*)key;
 - (NSMutableArray*)mutableArrayValueForKeyPath:(NSString*)keyPath;
-- (NSMutableSet*)mutableSetValueForKey:(NSString*)key STUB_METHOD;
-- (NSMutableSet*)mutableSetValueForKeyPath:(NSString*)keyPath STUB_METHOD;
-- (NSMutableOrderedSet*)mutableOrderedSetValueForKey:(NSString*)key STUB_METHOD;
-- (NSMutableOrderedSet*)mutableOrderedSetValueForKeyPath:(NSString*)keyPath STUB_METHOD;
+- (NSMutableSet*)mutableSetValueForKey:(NSString*)key;
```

Comment:
Whoops left this from when I arrogantly thought I could fit all of these changes in one PR 😈